docs/v0.21/testing-0.21rc2.md

```diff
@@ -96,33 +97,32 @@ $ tor
+The Bitcoin [testnet](https://en.bitcoin.it/wiki/Testnet) is a [proof-of-work](https://en.bitcoin.it/wiki/Proof_of_work) based testing framework where volunteers are relied on to [mine](https://en.bitcoin.it/wiki/Mining) [blocks](https://en.bitcoin.it/wiki/Block) with real CPU power and in turn receive worthless testnet coins. Since the economics of mainnet are not at play here, we get a network that is unpredictable and, frustratingly, unreliable.
 
 This release introduces [Signet](https://bitcoinops.org/en/topics/signet/), a new testing network. Signet does away with decentralized proof-of-work in favor of a centralized consensus mechanism where a group with authority is in charge of creating new blocks based on valid signatures. The aim is to create a testing network that is predictable and reliable.
 
 ## 1. Manual Testing
-The [Bitcoin Wiki](https://en.bitcoin.it/wiki/Main_Page) contains excellent documentation on connecting to and testing Signet. Follow this [guide](https://en.bitcoin.it/wiki/Signet) to test signet.
+The [Bitcoin Wiki](https://en.bitcoin.it/wiki/Main_Page) contains excellent documentation on connecting to and testing Signet. Follow this [guide](https://en.bitcoin.it/wiki/Signet) to test Signet.
 
 ---
 
 ## Testing Anchors
 An [eclipse attack](https://cs-people.bu.edu/heilman/eclipse/) is an attack on bitcoin's p2p network. In order for the attack to be effective, the attacker aims to restart your node and then supply your node with IP addresses controlled by the attacker. Eclipse attacks reduce the soundness of second layer solutions such as the lightning network.
 
-
-When you're node connects to the Bitcoin network, it makes at least [two outbound block-relay only connections](https://github.com/bitcoin/bitcoin/pull/15759). This release introduces [Anchor Connections](https://github.com/bitcoin/bitcoin/pull/17428). Anchors are the [two outbound block-relay connections]() your node is connected to; logged to an `anchors.dat` file so that they can be used upon a node restart. Under the assumption that you were connected to honest nodes before the attack, this aims to reduce an eclipse attack from being successful.
+When your node connects to the Bitcoin network, it makes at least [two outbound block-relay only connections](https://github.com/bitcoin/bitcoin/pull/15759). This release introduces [Anchor Connections](https://github.com/bitcoin/bitcoin/pull/17428). Anchors are the [two outbound block-relay connections]() your node is connected to, logged to an `anchors.dat` file so that they can be used upon a node restart. Under the assumption that you were connected to honest nodes before the attack, this aims to reduce an eclipse attack from being successful.
```

Comment:
I prefer the term block-relay-only connections. Full conns are also block-relay.